default_middlewares.go

```diff
@@ -0,0 +1,44 @@
+package fuego
+
+import (
+	"log/slog"
+	"net/http"
+)
+
+type LoggingConfig struct {
+	Disabled        bool
+	DisableRequest  bool
+	DisableResponse bool
+
+	RequestLogger  func(w http.ResponseWriter, r *http.Request)
+	ResponseLogger func(w http.ResponseWriter, r *http.Request)
+}
+
+var defaultLoggingConfig = LoggingConfig{
+	Disabled:       false,
```

Comment:
You can simply put this

```suggestion
```

Or you add the other Disable* one

Reply:
But maybe, the Disable should be a method checking if DisableRequest and DisableResponse leads to avoid using the middleware

Reply:
@ccoVeille good point, thank you. Disabled does feel a bit redundant.. it can just be inferred. have made some changes